### PR TITLE
Diagnostics tab reformat

### LINKED ISSUE #124: Diagnostics Tab Reformat
The goal of this is to move the graphs from the Diagnostics Tab to the Detailed Data page. The Diagnostics Tab will display Conditions and Predictions as they are determined. The Detailed Data pages will display nightly data. While the data to be displayed on the Detailed Data pages has to be finalized, this can be addressed later.

### PR DESCRIPTION
**Pull request by vinaynagappala**
Closes #124

### CODE REVIEW

**Review by williamchong256 — APPROVED** [review]

**Review by Firius0408 — CHANGES_REQUESTED** [review]
Blocked

**Comment by Firius0408** [comment]
@williamchong256 Please don't approve PRs without testing them yourself

File: frontend/src/detailedData.jsx [hunk 1]
```diff
@@ -90,11 +97,10 @@ function DetailedData({ navigation, route }) {
       })();
     }
   }, [item, value]));
-
   return (
   // Simple example of displaying data based in route
-    <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
```
Comment (Firius0408) [inline frontend/src/detailedData.jsx:96]: Why did you remove the `TouchableWithoutFeedback`? It is used to close the keyboard on iOS as introduced in #47

Comment (vinaynagappala) [reply frontend/src/detailedData.jsx:96]: yeah I talked to Courtney about this, and the keyboard closes without it on iOS and Android for me. I removed it because removing the feedback made the scroll not work

Comment (Firius0408) [reply frontend/src/detailedData.jsx:96]: Ok yeah that tests correct.

**Review by Firius0408 — APPROVED** [review]
